packages/vrender/src/graphic/graphic-service/graphic-service.ts

```diff
@@ -1305,14 +1305,14 @@ export class DefaultGraphicService implements IGraphicService {
     aabbBounds: IAABBBounds,
     graphic?: IGraphic
   ): IAABBBounds {
-    const {
-      outerRadius = arcTheme.outerRadius,
-      innerRadius = arcTheme.innerRadius,
-      endAngle = arcTheme.endAngle,
-      startAngle = arcTheme.startAngle
-    } = attribute;
+    const { outerRadius = arcTheme.outerRadius, innerRadius = arcTheme.innerRadius } = attribute;
+    let { endAngle = arcTheme.endAngle, startAngle = arcTheme.startAngle } = attribute;
 
-    // const { startAngle, endAngle } = this.getParsedAngle();
+    if (startAngle > endAngle) {
+      const temp = startAngle;
```

Comment:
we call use the next line to switch two vars:
```[startAngle, endAngle] = [endAngle, startAngle];```

Reply:
this  may cause performance issues because of array deconstruction in some packaging Configuration